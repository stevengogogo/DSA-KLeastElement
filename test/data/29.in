10 50000
8193 8248 -6285 -4098 2292 -5330 -5608 2485 -4632 -8947
Query 7 10 4
Query 10 10 1
Query 1 3 2
Reverse 4 5
Reverse 10 10
Query 2 7 3
Query 2 4 2
Delete 7
Reverse 3 8
Delete 7
Query 6 6 1
Reverse 5 8
Reverse 3 4
Query 4 8 5
Insert 5 -9589
Insert 9 -9745
Query 1 9 6
Insert 4 4104
Insert 9 2666
Insert 7 1650
Insert 6 -2407
Query 10 14 3
Query 6 10 5
Insert 15 -8968
Reverse 9 12
Delete 2
Reverse 11 14
Reverse 6 6
Query 9 13 1
Reverse 9 13
Delete 13
Delete 10
Query 8 9 1
Delete 3
Delete 5
Query 3 3 1
Query 6 6 1
Query 5 6 1
Query 1 2 1
Query 9 10 2
Reverse 8 9
Query 3 3 1
Reverse 5 5
Reverse 9 10
Delete 8
Reverse 9 9
Delete 3
Reverse 6 7
Delete 1
Reverse 3 6
Delete 3
Query 2 6 1
Query 4 4 1
Query 6 6 1
Insert 7 9855
Reverse 7 7
Delete 6
Insert 5 1608
Query 7 7 1
Query 3 3 1
Query 3 3 1
Delete 4
Reverse 1 6
Reverse 5 5
Reverse 4 4
Reverse 4 5
Insert 7 -2503
Reverse 7 7
Reverse 2 2
Query 2 4 3
Reverse 1 3
Reverse 7 7
Reverse 5 6
Delete 2
Reverse 5 5
Query 6 6 1
Delete 2
Delete 1
Query 1 1 1
Reverse 2 2
Reverse 4 4
Query 3 4 1
Delete 2
Insert 3 -5147
Insert 2 -1953
Insert 1 -8219
Insert 4 3683
Insert 1 -4751
Delete 3
Query 2 3 1
Insert 4 2590
Delete 6
Reverse 6 7
Reverse 2 2
Delete 4
Insert 2 7178
Delete 4
Query 3 3 1
Reverse 1 6
Query 1 2 2
Reverse 6 6
Insert 6 -462
Delete 1
Reverse 6 6
Reverse 3 6
Insert 3 6368
Reverse 1 1
Query 5 7 2
Delete 5
Delete 5
Query 1 5 5
Delete 2
Reverse 3 4
Reverse 4 4
Query 2 2 1
Delete 4
Delete 3
Delete 2
Query 1 1 1
Delete 1
