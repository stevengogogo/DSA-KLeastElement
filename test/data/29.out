2485
-8947
8193
-5330
2292
-4098
-4098
-4098
-5330
1650
-9745
-9745
-4632
-4098
-4098
2485
-6285
-4632
-8968
-4098
-8968
9855
-8947
-8947
1650
-2503
-2407
-8947
-8219
-8219
-2503
-462
6368
6368
-2503
